### DISCUSSION TITLE
Search in archives: results pane don't allow to do F3 in listbox for file in zip

### DISCUSSION THREAD

**Discussion by Alexey-T** [doublecmd/doublecmd#1072] [Suggestions]
### Description

Please support F3 (viewer) for search results listbox

![dc](https://user-images.githubusercontent.com/4154978/211913155-eb285024-d5fd-4519-a654-461dbf2c401d.png)

**Comment by PetbkA** [comment]
**1.1.0 beta Commit: cd3a69c65 x64 Windows** - guess it's working now, although archive is being opened in internal viewer even if you assign e.g. _7-Zip_ for this. See #716

**Comment by alexx2000** [comment]
No, it is still unsupported.
